src/containers/auth/login/index.js

```diff
@@ -51,15 +72,12 @@ class Login extends Component {
     }
   };
 
-  handleGoogleResponse = response => {
-    const { googleLoginRequest, onModalClose } = this.props;
-    googleLoginRequest(response);
-    onModalClose(false);
-  };
-
   render() {
     const { email, password, errors } = this.state;
     const { modalStatus } = this.props;
+    const linkedInURL = `api/users/auth/linkedin`;
```

Comment:
I don't know, but maybe, isn't a good idea, to make variables for google and linkedin urls? You can insert string presentetion of url to the href parameter of link right away. <`a href={'api/users/auth/google'}>`. First of all, you don't allocate memory for variable.